### DISCUSSION TITLE
Github Discussions. Worth using?

### DISCUSSION THREAD

**Discussion by dorkster** [flareteam/flare-game#847] [General]
I think we should try using this place as a replacement for the OpenGameArt forums. The current forum has its share of problems:

- Frequent errors. In fact, I can't even access the site through Chrome right now because the cert has expired.
- A bunch of posts from 2013-2015 were recently bumped to the top for no reason.
- Sometimes couldn't reply to a thread at all. My hypothesis is that it was because the parent deleted their account.
- Post formatting options are either limited or not as nice to use as Markdown. Especially when trying to post code.
- Occasional spam. Not much, but more than I think we'd get here.

Having a forum-like system here on Github should also help keep general questions out of the issue tracker.

I think it would be a good idea to only use the Discussions feature on this repo, but allow topics related to the engine. This repo is usually the one that gets linked to from other places, and has such became the "main" repo for the project.

**Comment by WithinAmnesia** [comment]
It could work.

**Comment by Danimal696** [comment]
It´s not a bad idea, we could center in here all of Flare know-how that´s spread into OGA. But the most important would be to try and stir more interest for Flare.

**Comment by Danimal696** [comment]
We could start by making a tutorial of all the setup that is needed to start using Flare. Since, if I had to format my pc right now, I would be  lost on how to properly restore everything again.

**Reply by dorkster** [reply]
Something like that should end up on the [wiki](https://github.com/flareteam/flare-engine/wiki). But there's no reason there can't be a discussion about creating it.

**Reply by WithinAmnesia** [reply]
Having a bunch of tutorials on how to make different areas and kinds of content for Flare would both make the current community stronger and make gaining more creative people easier and thus would be a good thing for new and old. Also I enjoy spell check here :-).

Archiving our methods and working skills and knowledge helps tremendously for community longevity. I remember the hell of doing modding archeology for Baldur's Gate since there were little to no accurate tutorials on how to make new content and all the old tools didn't work on new computers unless with convoluted compatibility shenanigans XD.

If we can document our methods and work it would avoid ~'legacy tool decay' (like using old versions of Blender) and would help new people get up to speed to enjoy current content and hopefully create more content; more inclusive the better. More the merrier.

**Reply by dorkster** [reply]
>like using old versions of Blender

This is probably a big one, since we used Blender's internal renderer for our models. I think that renderer disappeared after Blender 2.7, so none of our stuff renders correctly in new versions.

On Arch, there's a package for Blender 2.7 in the AUR, so it's easy to have both old and new installed side-by-side. But this isn't the case on a lot of other distros. If people don't want to mess with containers, their only option for Blender may be the newest version. So really, updating all our models to work with new Blender versions should definitely be somewhere on the roadmap.

**Reply by WithinAmnesia** [reply]
> 
> 
> > like using old versions of Blender
> 
> This is probably a big one, since we used Blender's internal renderer for our models. I think that renderer disappeared after Blender 2.7, so none of our stuff renders correctly in new versions.
> 
> On Arch, there's a package for Blender 2.7 in the AUR, so it's easy to have both old and new installed side-by-side. But this isn't the case on a lot of other distros. If people don't want to mess with containers, their only option for Blender may be the newest version. So really, updating all our models to work with new Blender versions should definitely be somewhere on the roadmap.

I am thinking if we update to newer renders in new (~current) Blender we should make a tutorial to show all of the render settings and set up like a scientific method so others can independently replicate the same results (possibly / hopefully standardized contiguous art style) created with the same tools. Also it would work as a tutorial. Also we can add in more monsters and assets if more people can work on animations and everyone wins. It is just getting more people to succeed via work to democratize the process / steps for success is a current issue with lack of documentation. For instance I don't even know how to properly animate even XD; but I am sure I can learn with a tutorial and a few pointers and tricks :-).

**Comment by MedicineStorm** [comment]
Is there still a link to the OGA forum within the engine/game itself?

Also, if the discussions are migrating here, would you like me to create a link from the OGA FLARE forum to here and/or lock the OGA forum?

**Reply by dorkster** [reply]
There's no links within the game, but the READMEs still link to the OGA forum. I can update this.

As long as the old posts are preserved, I think locking the OGA forum and linking to here would be a good idea.

**Reply by MedicineStorm** [reply]
Done. OGA FLARE forum is locked, but remains intact and searchable. Link to this discussion board is now shown in the FLARE forum description and a stickied migration announcement topic .
